.github/workflows/build.yaml

```diff
@@ -124,6 +124,16 @@ jobs:
       #       docker tag ghcr.io/predibase/lorax:main "$tag"
       #     done <<< "$tags"
 
+      - name: Run integration tests
```

Comment:
Ideally I would still want to push the image before integration tests run. This is important in cases where we need to get an image out for an urgent issue, etc.

What I would suggest: we have two separate actions for pushing main and pushing all other tags. The first action runs before int tests, the second runs after int tests. What do you think?

Reply:
agree!